pages/events/summer-school-2023.vue

```diff
@@ -222,6 +227,16 @@ const helpfulResourcesData = helpfulResources;
 
   &__cta {
     margin-top: carbon.$spacing-07;
+
+    &_disabled {
+      pointer-events: none;
+      color: qiskit.$text-color-light;
+      width: fit-content;
+
+      &:deep(.cta__icon) {
+        display: none;
+      }
+    }
```

Comment:
I'm not sure about how I feel about this implementation. Even though it does the job fine from a visual aspect, it renders kind of weird HTML in my opinion:

<img width="420" alt="Screenshot 2023-05-26 at 14 07 21" src="https://github.com/Qiskit/qiskit.org/assets/22047320/c7c7c45e-787e-45a8-accf-56a7b793f83c">

I think that `href="false"` and `rel="false"` are invalid values. This might confuse screen readers and other assistive technologies.

Overall I think that since this element's link attribute are being disabled, this should be just a "div" or "span" instead of a modified `UiCta` component.

Reply:
Interesting find @eddybrando - I didn't realize this before, and this is how we've been doing it for the past couple of years, _I think_.

I agree - we don't have to use the `UiCta`.

Reply:
After discussing with team, since this is such a short-lived change (will revert back in a week or less), we can go with this implemenation.